lib/xcodeproj/constants.rb

```diff
@@ -126,6 +126,7 @@ module Constants
     # @return [Hash] The compatibility version string for different object versions.
     #
     COMPATIBILITY_VERSION_BY_OBJECT_VERSION = {
+      52 => 'Xcode 11.0',
```

Comment:
can we actually load an object version 52 project?

Reply:
yes we can?

Reply:
I mean I successfully integrated and built an Xcode 11 project.

Reply:
Did you verify it had object version 52? Something must've changed in the project format, so there's got to be something else we need to update

Reply:
It is 52 and you are right https://github.com/CocoaPods/Xcodeproj/issues/688

Reply:
@segiddins done.